### ISSUE TITLE
[scroll-animations] Computed value of animation-timline

### ISSUE THREAD

**Issue by andruud** [w3c/csswg-drafts#5112]
The computed value of `animation-timeline` is currently "as specified". I think it should be as similar as possible to the computed value of `animation-name`, since it's identical (except `animation-timeline` also allows `auto`). It would appear `animation-name` isn't quite right in the spec either, as it doesn't handle string values vs. `<custom-ident>` correctly.

For `animation-name`, I think `<custom-ident>` is the canonical form, i.e. `animation-name:foo` rather than `animation-name:"foo"`, so probably the computed value for both properties should be a list, where each item is an identifier if it would match `<custom-ident>`, otherwise a string or a keyword.

In other words, if you specify `animation-name:"foo"` the computed value is `animation-name:foo`, but if you specify `animation-name:"none"`, the computed value is `animation-name:"none"`. And similarly for `animation-timeline`.

Alternatively make `animation-name` "as specified" too.

cc @majido

**Comment by andruud** [comment]
"As specified, but `<string>` values are replaced with case-sensitive _css identifiers_ if they would match `<custom-ident>`"

Something like that, maybe.

**Comment by majido** [comment]
I agree that we should match the `animation-name` semantic as close as possible here.

I don't have any particular opinion for the changes proposed for computed value of animation-name. Do you want to send a PR to css-animations to change the semantic of that and then we will update scroll-animations-1 to match that?

BTW, we should perhaps upstream animation-timeline to css-animations. Feel free to own that as well. :)

**Comment by majido** [comment]
I have upstreamed `animation-timeline` to css-animations-2 with its computed value matching `animation-name`.

**Comment by andruud** [comment]
Thanks, then that's consistent at least. We can revisit how animation-name/timeline is computed in a different issue against css-animations-2.
